data/core/node.lua

```diff
@@ -615,6 +615,12 @@ function Node:is_empty()
 end
 
 
+function Node:is_in_tab_area(x, y)
+  local _, ty, _, th = self:get_scroll_button_rect(1)
```

Comment:
This should probably check for `should_show_tabs`. At the moment even if tabs are hidden, the cursor changes to `arrow`.